.travis.yml

```diff
@@ -56,7 +56,6 @@ notifications:
           - http://julia.mit.edu:8000/travis-hook
 before_install:
     - make check-whitespace
-    - JULIA_SYSIMG_BUILD_FLAGS="--output-ji ../usr/lib/julia/sys.ji"
```

Comment:
It's also not what this test was written to test (since that's unnecessarily redundant and not quite correct). This was originally written to test the `--precompile=no` configuration, but was later mistakenly altered to its current form instead.